### DISCUSSION TITLE
[Feature Request] Link to Edit Page on GitHub

### DISCUSSION THREAD

**Discussion by joesan** [chipzoller/hugo-clarity#244] [Ideas]
While I was using Zola, I had made a few changes to the theme that I used so that I can have a link at the bottom of each page that shows a link to the original content in GitHub which upon click takes the user to the GitHub project page to be forked. 

As an example., here the learn theme has this GitHub link on all pages.

https://github.com/matcornic/hugo-theme-learn/blob/master/layouts/partials/header.html

Could this feature be added? It seems quite simple, but I'm not getting the settings. For example., here is the snippet that adds this to the pages:

```
                  <div id="top-github-link">
                    <a class="github-link" title='{{T "Edit-this-page"}}' href="{{ $.Site.Params.editURL }}{{ replace $.File.Dir "\\" "/" }}{{ $File.LogicalName }}" target="blank">
                      <i class="fas fa-code-branch"></i>
                      <span id="top-github-link-text">{{T "Edit-this-page"}}</span>
                    </a>
                  </div>
```
The steps should be to add this to the single.html just below this code snippet:

```
    <div class="post_body">
      {{- .Content }}
    </div>
```
Add the editURL to the params.toml. Correct?

**Comment by chipzoller** [comment]
@onweru

**Comment by joesan** [comment]
Any update on this?

**Reply by chipzoller** [reply]
Haven't worked on this but open to a PR.

**Comment by onweru** [comment]
I have created an issue. Will cover soon/.

**Reply by chipzoller** [reply]
Created #279
